test/integration.js

```diff
@@ -48,6 +48,19 @@ describe('c8', () => {
     )
   })
 
+  it('supports exeternally set NODE_V8_COVERAGE', () => {
+    process.env.NODE_V8_COVERAGE = './coverage/tmp_'
```

Comment:
Instead of overriding `NODE_V8_COVERAGE` of the whole process, use [`spawnSync()`](https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options) option to modify only this child process' environment.